src/Indice.Features.Cases.AspNetCore/Data/Models/DbCheckpointType.cs

```diff
@@ -7,17 +7,10 @@ public class DbCheckpointType
     {
         public Guid Id { get; set; }
         public Guid CaseTypeId { get; set; }
-        public string Code { get; private set; }
+        public string Name { get; set; }
```

Comment:
δεν βλέπω τον λογο να αλλάξουμε το code σε name! κωδικός είναι απλά τον τροπο που βάζουμε τιμές αλλάζουμε.

Reply:
Νέες αλλαγές, μετά την κουβέντα στο chat. Τις γράφω εδώ να τις ξαναδουμε ολοι

το model να γίνει 

```csharp
    public class DbCheckpointType
    {
        public Guid Id { get; set; }
        public Guid CaseTypeId { get; set; }
        public string Code { get; set; }
        public string? Title { get; set; }
        public string? Description { get; set; }
        public string? Translations { get; set; }
        public CasePublicStatus PublicStatus { get; set; }
        public bool Private { get; set; }
        public virtual DbCaseType CaseType { get; set; }
    }
```

Δηλαδή 
1. Κρατάμε το Code ως property για το business status (Rejected, Downloaded, etc)
2. να αποκτήσει Title & Translations. Νullable για την ώρα, δεν θα ασχοληθούμε. Απλά να έχουμε το ίδιο pattern με τα υπόλοιπα entities
2. Ubiquitous language -> Θα λέμε move to checkpoint "Code" και όχι "Name". Αφαιρουμε το name απ' το λεξιλόγιό μας :P